source/_integrations/generic_water_heater.markdown

```diff
@@ -0,0 +1,44 @@
+---
+title: Generic Water Heater
+description: Turn Home Assistant into a Water Heater
+ha_category:
+  - Water Heater
+ha_release: 2012.02
```

Comment:
```suggestion
ha_release: 2021.2
```

Reply:
Thank you for your review! 

Since PR have been running very slowly I'll wait for core PR approval before correcting this ;)